components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.spec.ts

```diff
@@ -24,4 +23,67 @@ describe('ProjectsDropdownComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('#projectsArray', () => {
+    it('#projectsArray returns an object naturally ordered by name', () => {
+      const unsortedProjects: {} = [
+        {
+          checked: false,
+          id: '123abc',
+          name: '123abc',
+          status: 'NO_RULES',
+          type: 'CUSTOM'
+        },
+        {
+          checked: false,
+          id: 'project-6',
+          name: 'project-6',
+          status: 'NO_RULES',
+          type: 'CUSTOM'
+        },
+        {
+          checked: false,
+          id: '12345zero',
+          name: '12345zero',
+          status: 'NO_RULES',
+          type: 'CUSTOM'
+        },
+        {
+          checked: false,
+          id: '123xyz',
+          name: '123xyz',
+          status: 'NO_RULES',
+          type: 'CUSTOM'
+        },
+        {
+          checked: false,
+          id: 'abc123',
+          name: 'abc123',
+          status: 'NO_RULES',
+          type: 'CUSTOM'
+        }
+      ];
+
+      // assign unsorted projects for the component to use
+      component.projects = unsortedProjects;
+
+      // expect sorted outcomes here
```

Comment:
These tests could be written more succinctly (and to give a more helpful error message if there's ever a failure), by doing something like:

```
actualProjects = component.projectsArray().map(p => p.name);
expect(actualProjects).toEqual(['bar', 'foo']);
```

Reply:
Smart - I just refactored it to do exactly that.  Thanks!